SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialPackageMapClient.h

```diff
@@ -62,6 +62,10 @@ class SPATIALGDK_API USpatialPackageMapClient : public UPackageMapClient
 
 	virtual bool SerializeObject(FArchive& Ar, UClass* InClass, UObject*& Obj, FNetworkGUID *OutNetGUID = NULL) override;
 
+	// Pending object references, being asynchronously loaded.
+	typedef TSet<FNetworkGUID> PendingRefSet;
```

Comment:
style nit - `using`

Reply:
Although does this need a typedef?

Reply:
Extra style nits - typedefs I think are ideally above function declarations. Also this is public for what seems like it wants to just be a private abbreviation.